src/backend/TrafficCourts/Citizen.Service/Services/Tickets/Search/Common/Invoice.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Text.Json.Serialization;
+
+namespace TrafficCourts.Citizen.Service.Services.Tickets.Search.Common
```

Comment:
Maybe this model object should be in a model namespace

Reply:
Invoice is an internal structure mapping to the structures returned by the RSI service. They are not and should not be exposed to the API.